src/routes/Evodex/BackLayer/Exchange/ExchangeBackLayer.js

```diff
@@ -88,6 +88,10 @@ const useStyles = makeStyles((theme) => ({
     alignItems: 'center',
     marginTop: theme.spacing(1),
     padding: theme.spacing(2, 0),
+    width: '70%',
```

Comment:
why reduce the width to 70% ? 

The exchange page looks off center and is not consistent 
![image](https://user-images.githubusercontent.com/5632966/92186898-71408880-ee15-11ea-93a6-1780b27d0864.png)

https://deploy-preview-151--evodex.netlify.app/exchange